curves/curve25519/Cargo.toml

```diff
@@ -13,6 +13,11 @@ edition = { workspace = true }
 bytemuck = { workspace = true }
 bytemuck_derive = { workspace = true }
 thiserror = { workspace = true }
+# this crate uses `subtle::CtOption<Scalar>::into_option` via curve25519-dalek,
+# which requires subtle v2.6.1, but curve25519-dalek only requires v2.3.0
+# The line below help users of this crate obtain correct subtle version.
+# (that is, the version specified by our workspace or greater minor version, not the version specified by curve25519-dalek)
+subtle = { workspace = true }
```

Comment:
Sorry, one more bit, the dependencies need to be sorted, so these lines need to go before `thiserror`

Reply:
okay.. done